.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
 Cargo.lock
+my_log_test*
```

Comment:
It's a little awkward to have tests that create files in the source directory. Would it be feasible to use `env!("OUT_DIR")` to obtain a directory to write the files into?

Reply:
Indeed it's not quite clean. There's no OUT_DIR env variable defined during testing, but instead we can use a temporary directory and redirect the process' cwd to this temporary directory. I've used the [tempfile](http://docs.rs/tempfile/3.0.5/tempfile/) crate as a dev dependency (so it's not added to the lib's users).

Reply:
Sounds good.